test/comparison_to_analytic_solutions.jl

```diff
@@ -24,6 +24,13 @@ struct DummyImpurityDensity{T} <: SolidStateDetectors.AbstractImpurityDensity{T}
     @testset "Capacity" begin
         @test all(isapprox.(C_ssd, C_Analytical, rtol = 0.001))
     end   
+    
+    @testset "Depletion Voltage" begin
+        dep_volt_1 = get_depletion_voltage(sim, 1, verbose = false)
+        dep_volt_2 = get_depletion_voltage(sim, 1, verbose = false)
+        @test isapprox(dep_volt_1, 0, atol = 0.2)
+        @test isapprox(dep_volt_2, 0, atol = 0.2)
+    end
```

Comment:
The depletion voltage is given by the impurity density of the crystal.
In this example, the detector has no impurity density.

Is this test to show that you can instantly deplete a detector if it has no impurities in it (a trivial test)?

Reply:
Yes, that was the goal